go/apps/mixnet/mixnet_test.go

```diff
@@ -110,6 +113,50 @@ func runProxyWriteCell(proxy *ProxyContext, msg []byte) error {
 	return nil
 }
 
+// Router accepts a connection from a proxy and handles two requests.
```

Comment:
Please give a little more detail. "handles two requests" makes it seem rather arbitrary when in fact it's handling a circuit creation message then a message for the circuit, correct?